testsuite/gateways/wasm/extension.py

```diff
@@ -212,6 +226,53 @@ def synchronise_credentials(self):
             return True
         return False
 
+    def add_authorities(self, authorities, wait=True):
```

Comment:
Couldn't it be fully replaced by `replace_authorities`?

Reply:
Could be. I parametrized the `replace_authorities` function